### DISCUSSION TITLE
Please make beatmap download settings savable in osu!lazer

### DISCUSSION THREAD

**Discussion by SUKEsann2000** [ppy/osu#31207] [Ideas] [answered]
**Now**
We have to remove featured artists and leaderboards every time on the download screen

**I want to**
Saving them

**Environment**
osu!lazer 2024.1219.2
Windows11

**Comment by peppy** [comment, marked as answer]
Featured artist setting is stored in the next release. Leaderboard and other filters should also be saved, but need further work to make it happen.

**Comment by SUKEsann2000** [comment]
Thank you!

**Comment by Siiiuuuuu** [comment]
I hope this gets added to the osu website aswell and not just ingame. 🙏
